linting/.skiptest

```diff
@@ -0,0 +1 @@
+Skips the current directory during automated test runs.
```

Comment:
@dan-mm
I added the `.skiptest` because the directory contains two usage scenario files and I'm pretty sure we don't have a way to handle that in our test setup right now.